tests/sessions_tests/tests.py

```diff
@@ -149,9 +149,6 @@ def test_clear(self):
         self.assertTrue(self.session.modified)
 
     def test_save(self):
-        if (hasattr(self.session, '_cache') and 'DummyCache' in
-                settings.CACHES[settings.SESSION_CACHE_ALIAS]['BACKEND']):
-            raise unittest.SkipTest("Session saving tests require a real cache backend")
```

Comment:
This was required when the `django.contrib.sessions.tests` were discovered by the old test runner.